actions/setup-yq/action.yml

```diff
@@ -0,0 +1,65 @@
+name: Install yq
+description: Sets up yq yaml parser - https://github.com/mikefarah/yq
+runs:
+  using: composite
+  steps:
+    - id: download-yq
+      shell: bash
+      run: |
+        set -e  # Exit on error
+        
+        # Configuration
+        ARCH="linux_amd64"
+        INSTALL_PATH="/usr/local/bin/yq"
+        
+        # Get latest version
+        VERSION=$(curl -sL https://api.github.com/repos/mikefarah/yq/releases/latest |
+        grep -o '"tag_name": *"[^"]*"' | cut -d'"' -f4 | sed 's/^v//')
+
+        echo "Latest version: $VERSION"
```

Comment:
Don't get latest. Specify the version to get as we do in setup-sops and setup-tenv